src/common/procedure/src/error.rs

```diff
@@ -28,6 +28,19 @@ use crate::PoisonKey;
 #[snafu(visibility(pub))]
 #[stack_trace_debug]
 pub enum Error {
+    #[snafu(display("Failed to procedure manager status"))]
```

Comment:
```suggestion
    #[snafu(display("Failed to check procedure manager status"))]
```

Reply:
resolved in [532432c](https://github.com/GreptimeTeam/greptimedb/pull/6393/commits/532432cfc4e8ef35c09d44ba1c089b6262273452)